src/DotNet/Writer/MethodBodyWriter.cs

```diff
@@ -58,29 +57,6 @@ public sealed class MethodBodyWriter : MethodBodyWriterBase {
 		/// </summary>
 		public uint LocalVarSigTok => localVarSigTok;
 
-		/// <summary>
-		/// Constructor
-		/// </summary>
-		/// <param name="helper">Helps this instance</param>
-		/// <param name="method">The method</param>
-		public MethodBodyWriter(ITokenProvider helper, MethodDef method)
```

Comment:
These two public constructors can't be removed, since it's a breaking change. Unless we wait to merge this PR so it's added in v4.0.

Deprecating them is an option. They're relatively new so should have few users.

Reply:
If wait v4, do we need merge IWriterError2 into IWriterError and delete Extensions.Error2 as breaking changes?

Reply:
We don't need to wait for v4 just keep both interface, but when v4 is released, these interfaces can be merged.